docker/diff/diff.py

```diff
@@ -119,17 +119,26 @@ def get_rows(diff_type: DiffType, cols: Dict[str, str], table_name: str = table_
              repo_data_project: str = repo_data_project, repo_dataset_name: str = repo_dataset_name,
              release_date: str = release_date, prev_date: str = prev_date, version_col: str = version_col,
              client: bigquery.Client = repo_client) -> str:
+    if table_name.lower() == "gene_association":
+        # if gene_association, then use these as ON vals
+        pk_cols = ["gene_id", "variation_id"]
```

Comment:
This kind of hardcoding makes me worry about fragility when new tables are added, or if we change the primary keys for a table. But I don't have an idea for a better way to structure it at this point.

Reply:
I agree; unfortunately, BigQuery has no notion of primary keys, so I can't ask it for the primary keys. I coulddd build this image off of the schema image and pull the primary keys that we define for the schema into this, that'd be more generalized/programmatically done.